packages/cyberstorm/src/components/Toast/Toast.module.css

```diff
@@ -0,0 +1,181 @@
+.root {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 0 12px 24px 5px rgb(0 0 0 / 0.65);
+}
+
+.contentWrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  border-left: 3px solid var(--feature-color);
+  border-radius: var(--border-radius--4);
+  background-color: var(--background-color);
+
+  --background-color: var(--alert-info-bg-color);
+  --feature-color: var(--alert-info-accent-color);
+  --text-color: var(--alert-info-text-color);
+}
+
+.content {
+  display: flex;
+  gap: var(--space--16);
+  align-items: flex-start;
+  align-self: stretch;
+  width: 100%;
+  padding: 1rem 3.25rem calc(1rem - 0.125rem) 1.5rem;
+}
+
+.closeIconWrapper {
+  position: absolute;
+  top: 0.5rem;
+  right: 0.5rem;
+  background-color: transparent;
+}
+
+.closeIcon {
+  display: flex;
+  align-items: center;
```

Comment:
I thought the Icon component was supposed to handle the centering and width/height?

Reply:
Old habits die hard